tools/internal/demo.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -e
+
+. tools/lib/lib.sh
+
+TUNNEL_PORT=${TUNNEL_PORT:-4242}
+SSH_KEY=${SSH_KEY:-~/.ssh/airbyte-app.pem}
```

Comment:
is this ssh key shared somewhere?

Reply:
now it is. In lastpass shared-engineering